moveit_configs_utils/moveit_configs_utils/moveit_configs_builder.py

```diff
@@ -345,15 +370,43 @@ def trajectory_execution(
         self.__moveit_configs.trajectory_execution = {
             "moveit_manage_controllers": moveit_manage_controllers,
         }
-        self.__moveit_configs.trajectory_execution.update(
-            load_yaml(
-                self._package_path
-                / (
-                    file_path
-                    or self.__config_dir_path / f"{self.__robot_name}_controllers.yaml"
+
+        # Find the most likely controller params as needed
+        if file_path is None:
+            config_folder = self._package_path / self.__config_dir_path
+            controller_pattern = re.compile("^(.*)_controllers.yaml$")
+            possible_names = get_pattern_matches(config_folder, controller_pattern)
+            if not possible_names:
+                raise RuntimeError(
+                    "trajectory_execution: `Parameter file_path is undefined "
+                    f"and no matches for {config_folder}/*_controllers.yaml"
                 )
-            )
-        )
+            else:
+                chosen_name = None
+                if len(possible_names) == 1:
+                    chosen_name = possible_names[0]
+                else:
+                    # Try a couple other common names, in order of precedence
+                    for name in [self.__robot_name, "ros", "ros2"]:
+                        if name in possible_names:
+                            chosen_name = name
+                            break
+                    else:
+                        option_str = "\n - ".join(
+                            name + "_controllers.yaml" for name in possible_names
+                        )
+                        raise RuntimeError(
+                            "trajectory_execution: "
+                            f"Unable to guess which parameter file to load. Options:\n - {option_str}"
+                        )
+                file_path = config_folder / (chosen_name + "_controllers.yaml")
+
+        else:
+            file_path = Path(file_path)
+            if not file_path.is_absolute():
+                file_path = self._package_path / file_path
```

Comment:
Is this needed? as far as I know, having `absolute path #1` / `absolute path #2` gives `absolute path #2` so I think using `self._package_path / file_path` directly below should be fine